### DISCUSSION TITLE
Having version issues.

### DISCUSSION THREAD

**Discussion by EmeraldGirl** [lllyasviel/Fooocus#2958] [Q&A] [answered]
ok so I am getting this when i launch~  I think i am supposed to add something to the bat file but I cant even find that.  Any help at all appreciated :)

Message I get... 

 When Fooocus 2.1.865 launches, it tells me I have Juggernaut6 and not8 and that I have to add a line.

You do not have [juggernautXL_v8Rundiffusion.safetensors] but you have [juggernautXL_version6Rundiffusion.safetensors].

Fooocus will use [juggernautXL_version6Rundiffusion.safetensors] to avoid downloading new models, but you are not using latest models.

Use --always-download-new-model to avoid fallback and always get new models.

**Comment by mashb1t** [comment, marked as answer]
This is not an issue but a notice, feel free to update the model whenever you want to.
If you've installed Fooocus using the zip file method, you can find the .bat files one level above the congfig.txt. If you like to always have the latest model, feel free to add `--always-download-new-model` to the run.bat, should then look like this (extending the default):

```
.\python_embeded\python.exe -s Fooocus\entry_with_update.py --always-download-new-model
pause
```

**Reply by EmeraldGirl** [reply]
So I had already gone ahead with deleting the whole thing from my PC and now I''m half way through downloading the ZIP file again, I got to ask, will reinstalling it automatically give me the latest  models ?  

And a big thank you for your previous reply :)

**Reply by mashb1t** [reply]
yes, it will, but that's the most bandwidth-intense option. I'd highly recommend adding the argument next time ^^

**Reply by EmeraldGirl** [reply]
Will do, thank you again :)
